index.html

```diff
@@ -1755,6 +1864,100 @@ <h2>
+      <p class="note">
+        [[HTML]]'s <code>registerProtocolHandler</code> method allows web sites
+        to register themselves as possible handlers for particular protocols.
+        What constitutes valid <code>protocol</code> and <code>url</code>
+        values for <a>ProtocolHandlerItem</a>s is defined in that API. Also
+        note that the [[HTML]] API uses <a href=
+        "https://url.spec.whatwg.org/#concept-url-scheme"><code>scheme</code></a>
+        where we use <code>protocol</code> but the same restrictions apply.
+      </p>
+      <section>
+        <h3>
+          `protocol` member
+        </h3>
+        <p>
+          The <dfn>protocol</dfn> member of a <a>ProtocolHandlerItem</a> is a
+          <a>string</a> that represents the protocol to be handled, such as
+          "mailto" or "web+auth".
+        </p>
+        <p>
+          The <a>protocol</a> member of a <a>ProtocolHandlerItem</a> is
+          equivalent to {{NavigatorContentUtils/registerProtocolHandler()}}'s
+          <a href="https://url.spec.whatwg.org/#concept-url-scheme">scheme</a>
+          argument defined in [[HTML]].
+        </p>
+      </section>
+      <section>
+        <h3>
+          `url` member
+        </h3>
+        <p>
+          The <dfn>url</dfn> member of a <a>ProtocolHandlerItem</a> is the
+          <a>URL</a> [=manifest/within scope=] of the application that opens
+          when the associated protocol is activated.
+        </p>
+        <p>
+          The <a>url</a> member of a <a>ProtocolHandlerItem</a> is equivalent
+          to {{NavigatorContentUtils/registerProtocolHandler()}}'s
+          <code>url</code> argument defined in [[HTML]].
+        </p>
+      </section>
+      <section>
+        <h3>
+          <dfn>Handling a protocol launch</dfn>
+        </h3>
+        <p>
+          When a <a>ProtocolHandlerItem</a> <var>protocol_handler</var> having
+          [=manifest=] <var>manifest</var> is invoked, it goes through the same
+          steps used to [=invoke a protocol handler=] defined in [=HTML=],
+          where the user agent SHOULD navigate to [=url=] and the appropriate
+          browsing context is set to a new top level browsing context.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Privacy and Security considerations
+        </h3>
+        <p>
+          Privacy concerns for custom scheme handling are detailed in the
+          <a href=
+          "https://html.spec.whatwg.org/multipage/system-state.html#security-and-privacy">
+          Security and privacy section</a> of
+          {{NavigatorContentUtils/registerProtocolHandler()}}.
+        </p>
+        <div class="note">
+          <p>
+            The user agent MUST ask for permission when using a protocol
+            handler for the first time. This feature requires user interaction
+            and a script cannot communicate with another application on its
+            own.
+          </p>
+        </div>
```

Comment:
Note: can't have a MUST inside a note.... either it's a normative requirement, or it's not.